README.md

```diff
@@ -115,6 +115,14 @@ You can deploy this project on github following instructions here:
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+## 🔭 Future Features <a name="future-features"></a>
+
+-  **Add details project page with live demo and viewing of source code**
+-  **Add Desktop platform features**
+-  **Customization of styles of the two platforms**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
 ## 🤝 Contributing <a name="contributing"></a>
 
```

Comment:
- [ ] I noticed that one of the links you included, specifically the M.I.T license link, is broken and no longer functional.
As you know, links are an essential part of any project, and it's crucial to ensure that they are functional and lead to the intended content. In this case, the broken link may hinder the ability of others to access important information related to your project. Kindly fix this :point_down: 
![Screenshot from 2023-04-28 00-11-25](https://user-images.githubusercontent.com/88894525/235003087-6b93b29d-d4e9-410a-8853-19e97fc006dc.png)